inspectors/models/abstractinspectormodel.cpp

```diff
@@ -264,6 +269,10 @@ QVariant AbstractInspectorModel::valueFromElementUnits(const Ms::Pid& pid, const
         return strList.join(",");
     }
 
+    case Ms::P_TYPE::INT: {
```

Comment:
Not sure whether it is needed. That was an attempt to fix the crash in slots processing.

Reply:
yes, this is not necessary, for simple cases like QVariant <int> or QVariant <QString> enough of what is in default